inst/shiny/ui.R

```diff
@@ -16,10 +16,14 @@ library(DT)
 # Name keys for human readable column names ----
 load("data/namekey.RData")
 
-citation <- paste0("W. Cowger, Z. Steinmetz, A. Gray, H. Hapich, C. Rochman, ",
-                   "J. Lynch, S. Primpke, K. Munno, H. De Frond, O. Herodotou. ",
-                   "2020. Open Specy v", packageVersion("OpenSpecy"),
-                   ". www.openspecy.org")
+pkgdesc <- packageDescription("OpenSpecy")
+citation <- paste(paste(
```

Comment:
I like this idea, but I think that making a citation like this will be hard for us to track how many citations we have which could be bad for out careers. Maybe we can recommend that people cite the exact version of the software and the publication that we will have coming out? @zsteinmetz

Reply:
Usually Google Scholar etc. can handle different versions quite well. To make this easier, I pulled the version information out of the "title field". Anyway, you're right that we should rather advertise citing the paper than the package once the paper is out. Yet, I found it quite useful to see which OS package version is running underneath the web app. So we might want to keep this bit once the paper is published.